README.md

```diff
@@ -91,7 +91,13 @@ $ brew install lima
 - On ARM hosts, QEMU has to be patched for enabling `--accel=hvf` support. See https://gist.github.com/nrjdalal/e70249bb5d2e9d844cc203fd11f74c55
 
 - Download the binary archive of Lima from https://github.com/lima-vm/lima/releases ,
-and extract it under `/usr/local` (or somewhere else).
+and extract it under `/usr/local` (or somewhere else). For instance, to extract it under `$HOME`:
+
+```bash
+cd ~
+VERSION=$(curl --silent "https://api.github.com/repos/lima-vm/lima/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | cut -c 2-)
```

Comment:
better to use jq

Reply:
If you use `jq` then you also need to mention `brew install jq`...

Reply:
> better to use jq

Thank you @AkihiroSuda! It's true, `sed` is ugly, while it assumed no dependencies as @jandubois suggested :-p